app/pages/apps/trivia/Trivia.jsx

```diff
@@ -1,14 +1,20 @@
-import React, { useReducer, useEffect } from "react";
+import React, { useReducer, useEffect, useRef, useState } from "react";
 import axios from "axios";
 import { connect } from "react-redux";
-import { Icon } from "@blueprintjs/core";
+import { Icon, Intent, Toaster, Position } from "@blueprintjs/core";
 import { fetchData } from "@datawheel/canon-core";
 import { Helmet } from "react-helmet-async";
 import Progress from "pages/apps/trivia/Progress";
 import Question from "pages/apps/trivia/Question";
 import Answers from "pages/apps/trivia/Answers";
 import TriviaContext from "pages/apps/trivia/TriviaContext.js";
 import TriviaReducer from "pages/apps/trivia/TriviaReducer.js";
+import DemographicForm from "../../game/birthle/components/DemographicForm/DemographicForm"
```

Comment:
Please use absolute paths instead of paths relative to the file... this makes it much easier to restructure and move files around in the future. For example this line should be rewritten as:

```
import DemographicForm from "pages/game/birthle/components/DemographicForm/DemographicForm"
```